### ISSUE TITLE
Could not find 'audacity-win-3.2.0-64bit'! (error 16)

### ISSUE THREAD

**Issue by tan-wei** [ScoopInstaller/Extras#9295]
```console
Updating 'audacity' (3.1.3 -> 3.2.0)
Downloading new version
audacity-win-3.2.0-64bit.zip (19.2 MB) [======================================================================] 100%
Checking hash of audacity-win-3.2.0-64bit.zip ... ok.
Uninstalling 'audacity' (3.1.3)
Removing shim 'Audacity.shim'.
Removing shim 'Audacity.exe'.
Removing shim 'Audacity.ps1'.
Unlinking ~\scoop\apps\audacity\current
Installing 'audacity' (3.2.0) [64bit] from extras bucket
Loading audacity-win-3.2.0-64bit.zip from cache
Extracting audacity-win-3.2.0-64bit.zip ... Could not find 'audacity-win-3.2.0-64bit'! (error 16)
所在位置 C:\Users\Winterreise\scoop\apps\scoop\current\lib\core.ps1:642 字符: 9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find ...it'! (error 16):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'audacity-win-3.2.0-64bit'! (error 16)
```

**Comment by OneYellowLemon** [comment]
Happening to me a well.

```
audacity: 3.1.3 -> 3.2.0
Updating one outdated app:
Updating 'audacity' (3.1.3 -> 3.2.0)
Downloading new version
Loading audacity-win-3.2.0-64bit.zip from cache
Checking hash of audacity-win-3.2.0-64bit.zip ... ok.
Uninstalling 'audacity' (3.1.3)
Installing 'audacity' (3.2.0) [64bit] from extras bucket
Loading audacity-win-3.2.0-64bit.zip from cache
Extracting audacity-win-3.2.0-64bit.zip ... Could not find 'audacity-win-3.2.0-64bit'! (error 16)
At C:\Users\<redacted>\scoop\apps\scoop\current\lib\core.ps1:642 char:9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find ...it'! (error 16):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'audacity-win-3.2.0-64bit'! (error 16)
```

**Comment by rasa** [comment]
Fixed in #9296
